packages/@react-types/tooltip/src/index.d.ts

```diff
@@ -37,7 +37,13 @@ export interface TooltipTriggerProps extends OverlayTriggerProps {
    * anchor element.
    * @default 0
    */
-  crossOffset?: number
+  crossOffset?: number,
+
+  /**
+   * By default, opens for both focus and hover. Can be made to open only for focus.
+   * @default 'default'
+   */
+  triggerAction?: 'default' | 'focus'
```

Comment:
Can we shorten this prop name to just `trigger`, and remove the `'default'` string? Usually we just omit the prop when we want the default.